packages/fx-core/tests/plugins/resource/frontend/unit/expectedBicepFiles/frontend_hosting_test.bicep

```diff
@@ -0,0 +1,22 @@
+@minLength(3)
+@maxLength(24)
+@description('Name of Storage Accounts for frontend hosting.')
+param frontend_hosting_storage_name string
+
+resource frontendHostingStorage 'Microsoft.Storage/storageAccounts@2021-04-01' = {
+  kind: 'StorageV2'
+  location: resourceGroup().location
+  name: frontend_hosting_storage_name
+  properties: {
+    accessTier: 'Hot'
+    supportsHttpsTrafficOnly: true
+    isHnsEnabled: false
+  }
+  sku: {
+    name: 'Standard_LRS'
+  }
+}
+
+output storageName string = frontendHostingStorage.name
+output endpoint string = frontendHostingStorage.properties.primaryEndpoints.web
+output domain string = replace(replace(frontendHostingStorage.properties.primaryEndpoints.web, 'https://', ''), '/', '')
```

Comment:
Seems the endpoint looks like `https://domain.com/`? Currently we output the endpoint without the tailing slash. It may break those plugins who consume the endpoint if we have it. Please help confirm.

Reply:
I have already trim the `https://` and the output is the same as current behavior. Thx.

Reply:
It seems this test bicep file is the same content as packages/fx-core/templates/plugins/resource/frontend/bicep/frontendHosting.bicep. Is it possible to use the same file in the unit test so that we can keep single source of truth?

Reply:
@eriolchan This file is to check frontend hosting plugin successfully generate bicep template files which can be correct bicep files after solution plugin compile it. Though some of the bicep template files have no changes during the process, we will keep the file here as a new file to show what the final wanted bicep file looks like.